docs/src/components/IconsList.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import {
+  Button,
+  Card,
+  CardSection,
+  Coupon,
+  Heading,
+  InputField,
+  Stack,
+  Text,
+  Tile,
+} from "@kiwicom/orbit-components";
+import { Link as LinkIcon, Search } from "@kiwicom/orbit-components/icons";
+import iconsList from "@kiwicom/orbit-components/lib/data/icons.json";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+
+import { CodeBlock } from "./Code";
+import { copyTimeout, pascalize } from "../utils/common";
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
+  grid-gap: ${({ theme }) => theme.orbit.spaceXSmall};
+`;
+
+const StyledAnchorWrapper = styled.a`
```

Comment:
In the console you have `index.js:2177 Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.`  Consider to have it like `div` or layout it differently.